dot-net/Centroid/Config.cs

```diff
@@ -78,7 +78,23 @@ public override bool TryConvert(ConvertBinder binder, out object result)
 
             try
             {
-                result = Convert.ChangeType(RawConfig, binder.Type);
+                JToken rawConfig = RawConfig;
+                result = rawConfig.ToObject(binder.Type);
+                //ensure at least one property in the config bound to the type
```

Comment:
Right now my workaround in projects to achieve this result is the Serialize and Deserialize using the Newtonsoft methods.  I'd like my usage of `Config` to not need to know about the inner workings of Newtonsoft.  I think that was kinda my initial intent of #64... (which obviously got kicked to the wayside)